common.sh

```diff
@@ -187,8 +187,11 @@ if [ "${UPSTREAM_IRONIC:-false}" != "false" ] ; then
     export BAREMETAL_OPERATOR_LOCAL_IMAGE=${BAREMETAL_OPERATOR_LOCAL_IMAGE:-"quay.io/metal3-io/baremetal-operator"}
 fi
 
+# Default installer command extracted
+export OPENSHIFT_INSTALLER_CMD=${OPENSHIFT_INSTALLER_CMD:-"openshift-baremetal-install"}
```

Comment:
Should it be `openshift-install`?

Reply:
Should be overridden in [03_build_agent_installer.sh](https://github.com/openshift-metal3/dev-scripts/pull/1393/files#diff-6e1794f1d1ee5f8fad62e52f1d304a71a79419e856594561a45ccf3efc3708ab), let me check it again